File: solutions/3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.py
class Solution:
    def canAliceWin(self, nums):
        dd, sd = 0, 0
        for i in nums:
            if i // 10 != 0:
                dd += i
            else:
                sd += i
        if abs(dd - sd) > 0:
            return True
        return False

File: solutions/3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int dd = 0, sd = 0;
        for(int i : nums) {
            if(i / 10 != 0) dd += i;
            else sd += i;
        }
        return Math.abs(dd - sd) > 0;
    }
}